### ISSUE TITLE
I'm not sure what I am doing wrong with the Clouds 1 Background

### ISSUE THREAD

**Issue by sykie** [tengbao/vanta#66]
I have all of my code in place like I was supposed to and the scripts I needed to download, but it still refuses to work. I don't know what I'm doing wrong, could I please have some help?

**Comment by Game3over** [comment]
I need help with that too.

**Comment by tengbao** [comment]
Could you please post a jsfiddle? Here is a working version: https://jsfiddle.net/4qbeko75/ @sykie @Game3over

**Comment by TanushN** [comment]
Its not working for me on Jsfiddle either.

**Comment by tarapichumani** [comment]
Hi @tengbao, I ran into the same issue. My javascript console gave me this error - looks like an issue with Three.js
```

THREE.WebGLProgram: shader error:  0 35715 false gl.getProgramInfoLog invalid shaders  THREE.WebGLShader: gl.getShaderInfoLog() fragment
ERROR: 0:125: '=' : global variable initializers must be constant expressions
```

**Comment by tarapichumani** [comment]
@tengbao Line 125 on vanta.clouds.min.js for me is 
`vec3 speed1 = vec3(0.5,0.01,1.0) * 0.5 * speed;
` inside of Effect.prototype.Shader
it worked if I replaced _speed_ with a hardcoded value (i.e. 1.0)

**Comment by tengbao** [comment]
Thanks for the bug report. Fixed in latest release vanta@0.5.18 - https://github.com/tengbao/vanta/commit/beb66e0f52a44437c480f66347ea9d415e7ce46f
